pkg/skaffold/build/buildpacks/lifecycle.go

```diff
@@ -47,6 +50,18 @@ func (b *Builder) build(ctx context.Context, out io.Writer, a *latest.Artifact,
 	artifact := a.BuildpackArtifact
 	workspace := a.Workspace
 
+	// Read `project.toml` if it exists.
+	var projectDescriptor Descriptor
+	path := filepath.Join(a.Workspace, projectTOML)
+	if _, err := os.Stat(path); err == nil {
```

Comment:
Sorry I don't understand what you'd like me to do

Reply:
I think this would work?
```
if projectDescriptor, err := ReadProjectDescriptor(path); err != nil && !is.IsNotExist(err) {
   return "", fmt.Errorf ...
}
```
?